src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/Commands.RecoveryServices.Backup.Cmdlets.csproj

```diff
@@ -106,7 +106,9 @@
     <Compile Include="RecoveryServicesBackupCmdletBase.cs" />
     <Compile Include="CmdletParameterHelpMessages.cs" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Content Include="AzureRM.RecoveryServices.Backup.dll-help.xml" />
```

Comment:
why not give name like this - Microsoft.Azure.Commands.RecoveryServices.Backup.dll-help.xml?